pkgs/by-name/nd/ndcurves/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  jrl-cmakemodules,
+  lib,
+  pinocchio,
+  pkg-config,
+  python3Packages,
+  pythonSupport ? false,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ndcurves";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "loco-3d";
+    repo = "ndcurves";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-XJ3VSSGKSJ+x3jc4408PGHTYg3nC7o/EeFnbKBELefs=";
+  };
+
+  outputs = [
+    "out"
+    "doc"
+  ];
+
+  strictDeps = true;
+
+  nativeBuildInputs =
+    [
+      cmake
+      doxygen
+      pkg-config
+    ]
+    ++ lib.optionals pythonSupport [
+      python3Packages.python
+      python3Packages.pythonImportsCheckHook
+    ];
+  propagatedBuildInputs =
+    [ jrl-cmakemodules ]
+    ++ lib.optionals pythonSupport [
+      python3Packages.eigenpy
+      python3Packages.pinocchio
+    ]
+    ++ lib.optional (!pythonSupport) pinocchio;
+
+  cmakeFlags =
+    [
+      (lib.cmakeBool "BUILD_PYTHON_INTERFACE" pythonSupport)
+      (lib.cmakeBool "CURVES_WITH_PINOCCHIO_SUPPORT" true)
+    ]
+    ++ lib.optional stdenv.hostPlatform.isDarwin (
+      lib.cmakeFeature "CMAKE_CTEST_ARGUMENTS" "--exclude-regex;'curves_tests|python-curves'"
+    );
+
+  doCheck = true;
```

Comment:
Tests fail on aarch64-linux and x86_64-darwin

Reply:
thanks, do you know which ones ? Or should I just avoid all tests on those platforms ?

Reply:
On x86_64-darwin: 8 - test-so3-smooth (Failed)
On aarch64-linux: 1 - curves_tests (Failed)

This is from the ofborg output. You can find it in the "Checks" tab.